### PR TITLE
Apache2

### PR DESCRIPTION
**Pull request by kuritka**
- bump to http://www.apache.org/licenses/LICENSE-2.0
- license management by [golic](https://github.com/kuritka/golic)

### CODE REVIEW

**Review by ytsarev — CHANGES_REQUESTED** [review]
If the headers were produced by https://github.com/kuritka/golic please somehow reference it?

File: .licignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Ignore everything
```
Comment (k0da) [inline .licignore:2]: What is the intention of this file?

**Comment by kuritka** [comment]
> If the headers were produced by https://github.com/kuritka/golic please somehow reference it?

@ytsarev , good idea! Will do. thx

File: .licignore (hunk 1 above)
Comment (kuritka) [reply .licignore:2]: @k0da , the purpose is to filter which files we inject the license. The file works the same way as `.gitignore`.

**Comment by k0da** [comment]
@kuritka then I agree with @ytsarev golic should be referenced somehow. Without reference and context, it looks like file is orphan.

**Comment by kuritka** [comment]
amended

**Review by k0da — APPROVED** [review]

**Review by ytsarev — APPROVED** [review]

**Pull request merged**